### PR TITLE
release instruction and script improvements

### PR DESCRIPTION
**Pull request by s-urbaniak**
### All Submissions:

* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

File: scripts/release-redhat-certified.sh [hunk 1]
```diff
@@ -15,15 +15,28 @@ pwd
...
   "${REPO}/${VERSION}"/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml
 
+# set containerImage annotation
+value="${IMG_SHA_AMD64}" yq e -i '.metadata.annotations.containerImage = "quay.io/mongodb/mongodb-atlas-kubernetes-operator@" + env(value)' \
+  "${REPO}/${VERSION}"/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml
+
+# set openshift versions
+yq e -i '.annotations = .annotations + { "com.redhat.openshift.versions": "v4.8" }' \
+  "${REPO}/${VERSION}"/metadata/annotations.yaml
+
```
Comment (s-urbaniak) [inline scripts/release-redhat-certified.sh:39]: Note: https://github.com/redhat-openshift-ecosystem/certified-operators/pull/3619 was done using this script.

**Review by josvazg — COMMENTED** [review]
These manual changes are still needed?

Eg:

> Before posting the PR there is a manual change you need to make:
>
> * Ensure to add the `quay.io/` prefix in all Operator image references.
> You can see an [example fixed PR here on Community Operators for version 1.9.1](https://github.com/k8s-operatorhub/community-operators/pull/3457).

I mean it has been fixed for **certified**, we could as well do it for the other 2.

**Review by josvazg — APPROVED** [review]
LGTM, I just wonder if we should also fix the other 2 redhat releases not to need to setup the `quay.io/` prefix manually.

**Review by helderjs — APPROVED** [review]

File: scripts/release-redhat.sh
Comment (helderjs) [inline scripts/release-redhat.sh:1]: Should it be done also in release-redhat-openshift.sh?

**Review by helderjs — CHANGES_REQUESTED** [review]

File: scripts/release-redhat.sh
Comment (s-urbaniak) [reply scripts/release-redhat.sh:1]: @helderjs no, `release-redhat-openshift.sh` just copies over stuff that is already touched by `release-redhat.sh`

**Review by helderjs — APPROVED** [review]

**Pull request merged**